geth/jail/internal/loop/loop_test.go

```diff
@@ -0,0 +1,67 @@
+package loop
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	gomock "github.com/golang/mock/gomock"
+
+	"github.com/status-im/status-go/geth/jail/internal/vm"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestLoopSuite(t *testing.T) {
```

Comment:
Top :+1: for adding a unit test. Could you please also add negative tests? E.g. with getting a timeout or when trying to add tasks to an already closed loop? Many thanks.

Reply:
There were negative tests. I separated them now so they are more obvious.